__tests__/app.test.js

```diff
@@ -261,4 +261,51 @@ describe("POST /api/articles/:article_id/comments",() => {
     expect(body.msg).toBe("Bad Request")
     });
   });
+});
+
+describe("PATCH /api/articles/:article_id",() => {
+  it("Should icrement or decrement the article votes by the given amount and respond with the updated aricle.", () => {
+    const newVotes = { inc_votes : 5 } 
+    return request(app)
+    .patch("/api/articles/7")
+    .send(newVotes)
+    .expect(200)
+    .then(({ body }) => {
+      const { updatedArticle } = body
+      expect(updatedArticle).toMatchObject(
+        {
+          article_id: 7,
+          title: 'Z',
+          topic: 'mitch',
+          author: 'icellusedkars',
+          body: 'I was hungry.',
+          created_at: '2020-01-07T14:08:00.000Z',
+          votes: 5,
+          article_img_url: 'https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700'
+        }
+      )
+    })
+  });
+
+  it("404: responds with an 'Inexistent Article' error message when given a valid but inexistent article", () => {
+    const newVotes = { inc_votes : 5 } 
+    return request(app)
+    .patch("/api/articles/5000")
+    .send(newVotes)
+    .expect(404)
+    .then(({ body }) => {
+    expect(body.msg).toBe("Inexistent Article")
+    });
+  });
+
+  it("400: responds with a 'Bad Request' error message when given a an invalid article id", () => {
+    const newVotes = { inc_votes : 5 } 
+    return request(app)
+    .patch("/api/articles/not-an-article")
+    .send(newVotes)
+    .expect(400)
+    .then(({ body }) => {
+    expect(body.msg).toBe("Bad Request")
+    });
+  });
```

Comment:
Great work. Just one more test please :) 
400: Bad request, incomplete body. E.g. if the inc_votes is not a number.